[Bug]: Typescript errors on Combobox

### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Combobox

### Expected behavior

No errors when compiling.

### Actual behavior

✘ [ERROR] TS2416: Property 'autocomplete' in type 'Combobox' is not assignable to the same property in base type 'Textfield'.
  Type '"list" | "none"' is not assignable to type 'AutoFill | undefined'.
    Type '"list"' is not assignable to type 'AutoFill | undefined'. [plugin angular-compiler]

    node_modules/@spectrum-web-components/combobox/src/Combobox.ts:65:20:
      65 │     public override autocomplete: 'list' | 'none' = 'none';
         ╵                     ~~~~~~~~~~~~


✘ [ERROR] TS2339: Property 'has' does not exist on type 'never'. [plugin angular-compiler]

    node_modules/@spectrum-web-components/combobox/src/Combobox.ts:503:20:
      503 │         if (changed.has('open')) {
          ╵                     ~~~


✘ [ERROR] TS2339: Property 'has' does not exist on type 'never'. [plugin angular-compiler]

    node_modules/@spectrum-web-components/combobox/src/Combobox.ts:509:20:
      509 │         if (changed.has('activeDescendant')) {
          ╵                     ~~~


✘ [ERROR] TS2339: Property 'get' does not exist on type 'never'. [plugin angular-compiler]

    node_modules/@spectrum-web-components/combobox/src/Combobox.ts:510:55:
      510 │             const previouslyActiveDescendant = changed.get(
          ╵                                                        ~~~


✘ [ERROR] TS2339: Property 'has' does not exist on type 'never'. [plugin angular-compiler]

    node_modules/@spectrum-web-components/combobox/src/Combobox.ts:524:20:
      524 │         if (changed.has('options') || changed.has('optionEls')) {
          ╵                     ~~~


✘ [ERROR] TS2339: Property 'has' does not exist on type 'never'. [plugin angular-compiler]

    node_modules/@spectrum-web-components/combobox/src/Combobox.ts:524:46:
      524 │         if (changed.has('options') || changed.has('optionEls')) {
          ╵                                               ~~~


✘ [ERROR] TS2488: Type 'HTMLCollection' must have a '[Symbol.iterator]()' method that returns an iterator. [plugin angular-compiler]

    node_modules/@spectrum-web-components/combobox/src/Combobox.ts:535:36:
      535 │ ...     const descendants = [...list.children] as SpectrumElement[];
          ╵                                 ~~~~~~~~~~~~~




### Screenshots

![image](https://github.com/adobe/spectrum-web-components/assets/3112165/535ea274-7740-4a96-ab75-eaa059462248)


### What browsers are you seeing the problem in?

_No response_

### How can we reproduce this issue?

npm install the bundle.
Attempt to ng-serve with latest angular.


### Sample code that illustrates the problem

_No response_

### Logs taken while reproducing problem

_No response_

--- a/packages/combobox/package.json
+++ b/packages/combobox/package.json
@@ -4,7 +4,7 @@
     "publishConfig": {
         "access": "public"
     },
-    "description": "",
+    "description": "Web component implementation of a Spectrum design Combobox",
     "license": "Apache-2.0",
     "repository": {
         "type": "git",
@@ -43,11 +43,12 @@
         "test": "echo \"Error: run tests from mono-repo root.\" && exit 1"
     },
     "files": [
-        "*.d.ts",
-        "*.js",
-        "*.js.map",
-        "/src/",
-        "custom-elements.json"
+        "**/*.d.ts",
+        "**/*.js",
+        "**/*.js.map",
+        "custom-elements.json",
+        "!stories/",
+        "!test/"
     ],
     "keywords": [
         "spectrum css",
@@ -73,6 +74,6 @@
     "customElements": "custom-elements.json",
     "sideEffects": [
         "./sp-*.js",
-        "./sp-*.ts"
+        "./**/*.dev.js"
     ]
 }

